<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgSettings</class>
 <widget class="QDialog" name="dlgSettings">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>366</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Game Settings</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QCheckBox" name="chkAdvanced">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>332</y>
     <width>211</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Enable advanced options&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="text">
    <string>Advanced Options</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>371</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tabGame">
    <attribute name="title">
     <string>Game</string>
    </attribute>
    <widget class="QComboBox" name="cboClient">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>86</y>
       <width>300</width>
       <height>33</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Game client version to use. 64-bit is the most modern. It does work with WINE</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblPatchClient">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>264</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Patch Client DLL</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtPatchClient">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>260</y>
       <width>300</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="btnGameDir">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>6</y>
       <width>36</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Select game directory from file system</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="lblGraphics">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>47</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enable high resolution game files. You may need to patch the game after enabling this</string>
     </property>
     <property name="text">
      <string>Hi-Res Graphics</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblClient">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>91</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Game client version to use. 64-bit is the most modern. It does work with WINE</string>
     </property>
     <property name="text">
      <string>Game Client</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lblGameDir">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>10</y>
       <width>131</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Game install directory. There should be a file called patchclient.dll here</string>
     </property>
     <property name="text">
      <string>Game Directory</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtGameDir">
     <property name="geometry">
      <rect>
       <x>151</x>
       <y>5</y>
       <width>260</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Game install directory. There should be a file called patchclient.dll here</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btnSetupWizard">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>150</y>
       <width>301</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Run Setup Wizard</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="highResCheckBox">
     <property name="geometry">
      <rect>
       <x>155</x>
       <y>45</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enable high resolution game files. You may need to patch the game after enabling this</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabWine">
    <attribute name="title">
     <string>Wine</string>
    </attribute>
    <widget class="QGroupBox" name="wineFormGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>18</y>
       <width>451</width>
       <height>151</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enable manual WINE configuration. Otherwise, WINE and DXVK all get downloaded and managed automatically</string>
     </property>
     <property name="title">
      <string>Manually Configure</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="flat">
      <bool>true</bool>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
     <layout class="QFormLayout" name="wineGroupBox">
      <property name="formAlignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="leftMargin">
       <number>15</number>
      </property>
      <property name="topMargin">
       <number>15</number>
      </property>
      <property name="rightMargin">
       <number>15</number>
      </property>
      <property name="bottomMargin">
       <number>15</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="prefixLabel">
        <property name="toolTip">
         <string>Path to WINE prefix</string>
        </property>
        <property name="text">
         <string>Prefix</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="prefixLineEdit">
        <property name="toolTip">
         <string>Path to WINE prefix</string>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="wineExecutableLabel">
        <property name="toolTip">
         <string>Path to WINE executable</string>
        </property>
        <property name="text">
         <string>Executable</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="wineExecutableLineEdit">
        <property name="toolTip">
         <string>Path to WINE executable</string>
        </property>
        <property name="dragEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="wineAdvancedFrame">
     <property name="geometry">
      <rect>
       <x>-1</x>
       <y>159</y>
       <width>461</width>
       <height>141</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>461</width>
        <height>131</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="wineAdvancedForm">
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="wineDebugLabel">
         <property name="toolTip">
          <string>Value for WINEDEBUG environment variable</string>
         </property>
         <property name="text">
          <string>WINEDEBUG</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="wineDebugLineEdit">
         <property name="toolTip">
          <string>Value for WINEDEBUG environment variable</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <zorder>wineAdvancedFrame</zorder>
    <zorder>wineFormGroupBox</zorder>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="btnBoxOptions">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>332</y>
     <width>450</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>2</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QRadioButton" name="btnEN">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Language unavailable. It must be selected in the official launcher, so that the client_local_English.dat file can be downloaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnDE">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Language unavailable. It must be selected in the official launcher, so that the client_local_DE.dat file can be downloaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnFR">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Language unavailable. It must be selected in the official launcher, so that the client_local_FR.dat file can be downloaded.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>tabWidget</zorder>
  <zorder>btnBoxOptions</zorder>
  <zorder>chkAdvanced</zorder>
  <zorder>horizontalLayoutWidget</zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>accepted()</signal>
   <receiver>dlgSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoxOptions</sender>
   <signal>rejected()</signal>
   <receiver>dlgSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
